amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -110,22 +131,25 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
       )
     }
 
-    const { pageIndex, popularTables, searchResults, searchTerm, totalResults } = this.props;
-    const showResultsList = searchResults.length > 0 || popularTables.length > 0;
+    const items = this.props.tables;
+    const { page_index, results, total_results } = items;
+    const { popularTables } = this.props;
+
+    const showResultsList = results.length > 0 || popularTables.length > 0;
 
     if (showResultsList) {
-      const startIndex = (RESULTS_PER_PAGE * pageIndex) + 1;
-      const endIndex = RESULTS_PER_PAGE * ( pageIndex + 1);
-      const hasSearchResults = totalResults > 0;
+      const startIndex = (RESULTS_PER_PAGE * page_index) + 1;
+      const endIndex = RESULTS_PER_PAGE * ( page_index + 1);
+      const hasSearchResults = total_results > 0;
       const listTitle = hasSearchResults ?
-        `${startIndex}-${Math.min(endIndex, totalResults)} of ${totalResults} results` :
+        `${startIndex}-${Math.min(endIndex, total_results)} of ${total_results} results` :
```

Comment:
This code block has at least 3 times a same condition `hasSearchResults`, and would be cleaner if you make a separate block for that condition instead of inline conditions in multiple places. 
And based on that condition you can also identify what are the results to be passed to `SearchList` in the `return()`